apps/api/src/common/services/email.service.ts

```diff
@@ -63,38 +63,20 @@ export class EmailService {
   private async isEmailUnsubscribed(userEmail: string): Promise<boolean> {
     if (!this.resendAudienceId) return false;
     try {
-      // Try SDK list with an email filter if supported
+      // Fetch a page of contacts for the audience and search locally by email
       const lower = userEmail.toLowerCase();
-      const maybeListWithFilter: any = await (this.resend as any).contacts
-        ?.list?.({
-          audienceId: this.resendAudienceId,
-          email: userEmail,
-          limit: 200,
-        })
-        .catch(() => null);
-
-      const extractContacts = (result: any): any[] => {
-        if (!result) return [];
-        // Resend SDK/result shapes vary; handle a few possibilities safely
-        if (Array.isArray(result)) return result;
-        if (Array.isArray(result?.data)) return result.data;
-        if (Array.isArray(result?.data?.data)) return result.data.data;
-        if (Array.isArray(result?.results)) return result.results;
-        return [];
-      };
+      const list = await this.resend.contacts.list({
```

Comment:
@CharlieHelps is there a way in the API / SDK to look up just a single email address rather than looking up the entire list every time?

If not, can you make sure we paginate here?